test/test.js

```diff
@@ -46,3 +46,19 @@ describe('isOccurrencesValid function returns false on obviously false numbers',
     assert.deepStrictEqual(result, false)
   })
 })
+
+describe('Generate unique tax ids', () => {
+  it('Generates the number of tax ids provided', () => {
+    const taxIdArray = generateUniqueSteuerIds(2)
+    assert.strictEqual(taxIdArray.length, 2)
+  })
+  it('Generates valid tax ids', () => {
+    const taxIdArray = generateUniqueSteuerIds(2)
+    taxIdArray.forEach((id) => assert.strictEqual(isSteuerIdValid(id), true))
+  })
+  it('Generates a unique set of tax ids', () => {
+    const taxIdArray = generateUniqueSteuerIds(10)
+    const taxIdSet = new Set(taxIdArray)
+    assert.strictEqual(taxIdArray.length, taxIdSet.size)
```

Comment:
```suggestion
    const length = 10;
    const taxIdArray = generateUniqueSteuerIds(length)
    assert.strictEqual(taxIdArray.length, length)
```

Reply:
this is meant evaluate the uniqueness of the values rather than just the size of the array.

Reply:
Thanks for clarifying